### ISSUE TITLE
1991 Ewald not synchronised

### ISSUE THREAD

**Issue by sebfar9172** [nmathey/finasync#13]
Hello
I have 2 token of 1991 Ewald but these tokens are not synchronised

**Comment by nmathey** [comment]
Hi,
It does work on my test: address format is ok and this token has all info needed for Finary from the RealT API.
<img width="538" alt="Capture d’écran 2023-11-05 à 09 11 36" src="https://github.com/nmathey/finasync/assets/20896232/59daf755-4572-4d40-899c-24b0557aff46">

Try run the sync command again, maybe the Finary API was not available just for this property integration (?)

**Comment by sebfar9172** [comment]
Hello
https://gnosis.blockscout.com/api?module=account&action=tokenlist&address= don't return all the token realt like 1991 Ewald but many others

**Comment by nmathey** [comment]
yeah it happen sometimes, as block scout is not viable 100% of the time, this is why I created #9 but doesn't look that simple apparently.
Try rerun Finasync again later and see if it work.
Getting multiple other token is a problem I covered, so if it return a realty token I get the proper info, but it has to return a RealT token at least :)
